index.js

```diff
@@ -0,0 +1,65 @@
+const navBtn = document.querySelector('.nav-btn');
+const mobileNav = document.querySelector('.navbar');
+const speakersElm = document.getElementById('speakers');
+
+navBtn.addEventListener('click', () => {
+  mobileNav.classList.toggle('show');
+  if (mobileNav.classList.contains('show')) {
+    navBtn.innerHTML = '<i style="font-size: 30px;" class="fa-solid fa-xmark fa-2x"></i>';
+  } else {
+    navBtn.innerHTML = '<i class="fa-solid fa-bars fa-2x"></i>';
+  }
+});
+
+const speaker = [
+  {
+    speakerImg: 'image/speaker1.JPG',
+    speakerName: 'Yochai Benkler',
+    speakerJob: 'Berkman Professor of Enterpernieal legal studies at Harvard law School',
+    speakerInfo: 'Benkler studies commans-based peer production, and published his seminal book the Wealth of Networks in 2006',
+  },
+  {
+    speakerImg: 'image/speaker1.JPG',
+    speakerName: 'Yochai Benkler',
+    speakerJob: 'Berkman Professor of Enterpernieal legal studies at Harvard law School',
+    speakerInfo: 'Benkler studies commans-based peer production, and published his seminal book the Wealth of Networks in 2006',
+  },
+  {
+    speakerImg: 'image/speaker1.JPG',
+    speakerName: 'Yochai Benkler',
+    speakerJob: 'Berkman Professor of Enterpernieal legal studies at Harvard law School',
+    speakerInfo: 'Benkler studies commans-based peer production, and published his seminal book the Wealth of Networks in 2006',
+  },
```

Comment:
- [ ] Please add different content for each speaker because it will make your project more authentic and nice to look at when a recruiter goes through your project.